CONTRIBUTING.md

```diff
@@ -95,6 +95,22 @@ Try to keep the changes focused. If you submit a large amount of work in all in
 
 #### 4. Submit a [pull request](https://help.github.com/articles/about-pull-requests/)
 
+Please keep the title of your pull request short but informative - it will 
+appear in the changelog.
+
+Use one of the following prefixes in the title of your pull request:
+  - `[ENH]` - enhancement of the specification that adds a new feature or 
+    support for a new data type
+  - `[FIX]` - fix of a typo or language clarification
+  - `[INFRA]` - changes to the infrastructure automating the specification 
+    release (for example building HTML docs etc.)
+  - `[MISC]` - everything else including changes to the file listing 
+    contributors
+
+If you are opening a pull request to obtain early feedback, but the changes 
+are not ready to be merged (a.k.a. Work in Progress pull request) please 
+use [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+
 A member of the BIDS Specification team will review your changes to confirm that they can be merged into the main codebase.
```

Comment:
Maybe add a sentence to the effect of: "Reviewers have discretion to edit your title before merging." The idea is to let users know that it's not on them to get it perfect on the first try, and also not to be surprised if someone changes it.

Reply:
Good point, but modifying this part of the guide is not the focus of this PR. Would you like to open a PR/issue?